src/ir/ir_traversing_visitor_test.cc

```diff
@@ -151,6 +154,151 @@ TEST(IRTraversingVisitorTest, TraversesOperationAsExpected) {
       testing::ElementsAre(std::addressof(plus_op_instance), plus_op_module,
                            plus_op_module, std::addressof(plus_op_instance)));
 }
+}  // namespace
+
+namespace {
+using ResultType=std::vector<void*>;
+
+// Another small test visitor that collects the nodes as they are visited to make
+// sure that  Pre and Post visits happened correctly.
+// Uses 'returns' rather than side effects.
+class ReturningVisitor : public IRTraversingVisitor<ReturningVisitor, ResultType> {
+ public:
+  enum class TraversalType { PRE = 0x1, POST = 0x2, BOTH = 0x3 };
```

Comment:
According to the C++ style guide, enumerator names should be of the form `kEnumeratorName`: https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
Thanks, this is copied code, would you like me to rename both my copy and the original?

Reply:
Fixed both :)